tests/test_utils.py

```diff
@@ -345,26 +352,56 @@ def test_load_save_jsonl(tmpdir):
     assert li == lo
 
 
-def test_load_ee_models_library(tmpdir, monkeypatch):
-    fake_root_path = pathlib.Path(str(tmpdir)) / "data_and_models"
-
-    # Create directory structure and files
-    original_df = pd.DataFrame(
-        {"entity_type": ["A"], "model": ["model_1"], "entity_type_name": ["B"]}
-    )
-
-    csv_path = fake_root_path / "pipelines" / "ner" / "ee_models_library.csv"
-    csv_path.parent.mkdir(parents=True)
-    original_df.to_csv(csv_path)
-
-    df = load_ee_models_library(fake_root_path)
-
-    # Checks
-    assert isinstance(df, pd.DataFrame)
-    assert df["entity_type"][0] == "A"
-    assert df["model_path"][0] == str(fake_root_path / "models" / "ner_er" / "model_1")
-    assert df["model_id"][0] == "data_and_models/models/ner_er/model_1"
-    assert df["entity_type_name"][0] == "B"
+@pytest.mark.parametrize(
+    "metadata,expected_value",
+    [
+        ({"wrongkey": "wrongvalue"}, False),  # labels key missing
+        ({"labels": "wrongvalue"}, False),  # ner key missing
+        ({"labels": {"ner": ["SEVERAL", "ENTITIES"]}}, False),  # several entities
+        ({"labels": {"ner": ["organism"]}}, False),  # entity lower
+        ({"labels": {"ner": ["ORGANISM"]}}, True),  # Good structure
+    ],
+)
+def test_check_entity_type_consistency(tmpdir, metadata, expected_value):
+    """Test that checks are working"""
+    # Wrong model directory name
+    wrong_model_dir = pathlib.Path(tmpdir) / "models" / "ner_er" / "wrongmodelname"
+    wrong_model_dir.mkdir(parents=True)
+    assert check_entity_type_consistency(wrong_model_dir) is False
+
+    # Missing meta.json file
+    good_model_dir = pathlib.Path(tmpdir) / "models" / "ner_er" / "model-organism"
+    good_model_dir.mkdir(parents=True)
+    assert check_entity_type_consistency(good_model_dir) is False
+
+    # meta.json structure
+    meta_path = good_model_dir / "meta.json"
+    with open(meta_path, "w") as f:
+        f.write(json.dumps(metadata))
```

Comment:
Same as above, one can write `json.dump(metadata, f)`

Reply:
Done in 7f5c0d6.